pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -310,6 +321,12 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *CAOptions) (
 		rootCertFile = ""
 	}
 
+	if customca.IsUsingCustomCAForWorkloadCertificate(s.environment.Mesh().GetCa()) {
+		// In case: Istiod using Custom CA to signing Workload Certificates
+		// RootCAs will contains 2 rootCAs: one self signed, one from Custom CA
+		rootCertFile = s.environment.Mesh().GetCa().TlsSettings.CaCertificates
```

Comment:
if TlsSettings is not set this will get a nil pointer?

Reply:
![image](https://user-images.githubusercontent.com/49465776/95414123-4b982900-0957-11eb-9737-4d2b822e3086.png)